Extension/src/LanguageServer/extension.ts

```diff
@@ -184,7 +186,7 @@ export async function activate(): Promise<void> {
 
     disposables.push(vscode.workspace.onDidChangeConfiguration(onDidChangeSettings));
     disposables.push(vscode.window.onDidChangeActiveTextEditor(onDidChangeActiveTextEditor));
-    ui.activeDocumentChanged(); // Handle already active documents (for non-cpp files that we don't register didOpen).
+    if (ui instanceof OldUI) { void ui?.activeDocumentChanged(); } // Handle already active documents (for non-cpp files that we don't register didOpen).
```

Comment:
Please format these `if (ui instanceof OldUI)` lines. We don't normally put full blocks on a single line in TypeScript.